Tests/SwiftCliCoreTests/NestedArrayBuilderTest.swift

```diff
@@ -15,6 +15,16 @@ class NestedArrayMakerTest: XCTestCase {
         expect(result).to(equal(["k1", "k1", "k1"]))
     }
 
+    func testNestedArrayBuilderWithRepeats() throws {
+        let result = NestedArrayBuilder().arrayMaker(row: "k1 p1 (2x)")
+        expect(result).to(equal(["k1", "p1", "k1", "p1"]))
+    }
+
+    func testNestedArrayBuilderWithRepeatsMidRow() throws {
+        let result = NestedArrayBuilder().arrayMaker(row: "k1 p1 (2x) yo k1")
+        expect(result).to(equal(["k1", "p1", "k1", "p1", "yo", "k1"]))
+    }
```

Comment:
Some missing test cases:

multiple repeats in row: "k1 p1 (3x) k1 yo k1 (2x) k1 p1 (3x)"
single repeat: "k1 p1 (1x) k15" <-- I think you do sometimes see this in a pattern so let's support it
zero repeat: "k1 p1 (0x)" <-- probably should be an error
single stitch in repeat: "k1 (15x)" <-- should this be supported or should the user be prompted to put in "k15" instead? 🤔

Reply:
For the "k1 p1 (1x) k15" test case, the NestedArrayBuilder only deals with the (1x) repeat and the k15 gets "expanded" later. This feels like a code smell.... 🤔

Reply:
Oh interesting. Well, I think it's not a blocker for this PR since the `NestedArrayBuilder` _can_ handle the (1x), but let's discuss further at IPM.